package.json

```diff
@@ -101,8 +101,6 @@
     "prettier": "2.6.2",
     "query-string": "^7.0.1",
     "react-devtools-core": "4.24.7",
-    "react-dom": "18.0.0",
-    "react-native-web": "^0.18.7",
```

Comment:
thanks for removing these. I've added a [Trello ticket](https://trello.com/c/rSOcS3tM/18-remove-unused-non-expo-related-code-and-packages) to clear out unused/non-expo related code & packages.